cmd/podman/varlink/io.podman.varlink

```diff
@@ -699,7 +699,7 @@ method GetAttachSockets(name: string) -> (sockets: Sockets)
 # or name, a [ContainerNotFound](#ContainerNotFound) error is returned.
 method WaitContainer(name: string, interval: int) -> (exitcode: int)
 
-# RemoveContainer requires the name or ID of container as well a boolean representing whether a running container can be stopped and removed, and a boolean
+# RemoveContainer requires the name or ID of container as well an int as to whether a running container can be stopped and removed, and a boolean
```

Comment:
suggest: "RemoveContainer requires the name or ID of a container as well as an int that indicates whether a running container can be stopped and removed, ..."